crates/matrix-sdk-base/Cargo.toml

```diff
@@ -53,13 +54,18 @@ ruma = { version = "0.6.1", features = ["client-api-c", "js", "signatures"] }
 ruma = { version = "0.6.1", features = ["client-api-c", "signatures"] }
 
 [dev-dependencies]
-futures = { version = "0.3.21", default-features = false, features = ["executor"] }
+futures = { version = "0.3.21", default-features = false, features = [
```

Comment:
This isn't related to the change, I guess `cargo add` or some similar thing made these formatting changes?